libsodium-sys/Cargo.toml

```diff
@@ -32,7 +32,7 @@ libflate = "0.1"
 tar = "0.4"
 
 [dependencies]
-libc = { version = "=0.2.48" , default-features = false }
+libc = { version = "0.2.48" , default-features = false }
```

Comment:
we don't need to pin to this version. we can go with "0.2"

Reply:
Why? Let it be

Reply:
This doesn't pin it, but defines a minimum version. Should I change it regardless?

Reply:
this could still cause conflicts or have multiple version if a crate uses libc < 0.2.48. So unless there is a good reason to pin to 0.2.48 (such as a feature that was introduced in that release or a fix) it is better to be more "liberal" by specifying 0.2

Reply:
ouch we were both typing at the same time :P well yeah not a "pin" but a lower limit, but it just reduces the versions that could collide. I'd prefer 0.2 but either way i'm good with it